Sources/StreamChatUI/MessageActionsPopup/ChatMessagePopupVC.swift

```diff
@@ -25,24 +35,36 @@ open class _ChatMessagePopupVC<ExtraData: ExtraDataTypes>: _ViewController, UICo
             .withoutAutoresizingMaskConstraints
     }()
 
-    public private(set) lazy var messageContentView = uiConfig.messageList.messageContentView.init()
+    /// New instance of `messageContentViewClass` that is populated with `message` data
+    open private(set) lazy var messageContentView = messageContentViewClass.init()
         .withoutAutoresizingMaskConstraints
 
+    /// `messageContentView` class that is populated with `message` and shown
+    public var messageContentViewClass: _ChatMessageContentView<ExtraData>.Type!
+    /// Message data that is shown
     public var message: _ChatMessageGroupPart<ExtraData>!
+    /// Initial frame of a message
     public var messageViewFrame: CGRect!
-    public var originalMessageView: UIView!
+    /// `_ChatMessageActionsVC` instance for showing actions
     public var actionsController: _ChatMessageActionsVC<ExtraData>!
+    /// `_ChatMessageReactionsVC` instance for showing reactions.
     public var reactionsController: _ChatMessageReactionsVC<ExtraData>?
-
-    // MARK: - Private
-
-    private var actionsView: UIView { actionsController.view }
-    private var actionsViewHeight: CGFloat { CGFloat(actionsController.messageActions.count) * 40 }
-    private var reactionsView: UIView? { reactionsController?.view }
-    private var reactionsViewHeight: CGFloat { reactionsView == nil ? 0 : 40 }
-    private var spacing: CGFloat = 8
-
-    // MARK: - Life Cycle
+    
+    /// Spacing between reactions, message, and actions
+    public var spacing: CGFloat = 8
```

Comment:
`constraint(greaterThanOrEqualToSystemSpacingAfter:multiplier:)` ;-)

Reply:
removed this altogether since the spacing is handled by `ContainerView`